### ISSUE TITLE
Select2 ajax in custom widget

### ISSUE THREAD

**Issue by RudyCity** [elementor/elementor#14498]
i create a custom widget with control select2 where is use ajax option. and its work with ajax results, but i facing problems when i click another section and back to widget. select2 field not show the past result where has choosen.

this my code:
....
          $repeater->add_control(
                'select_product_product',
                [
                    'label' => "Choose Product",
                    'type' => Controls_Manager::SELECT2,
                    'label_block' => true,
                    'select2options' => [
                        'placeholder' => "Write Title Product",
                        'ajax' => [
                            'url'       => admin_url( 'admin-ajax.php' ) . "?action=product_search_elementor",
                            'dataType'  => 'json',
                            'delay'     => 250,
                            'cache'     => 'true'
                        ],
                        'minimumInputLength' => 3
                    ]
                ]
            );
....

and this is ajax response return :
$search_results = new WP_Query( array( 
                's'=> $q,
                'ignore_sticky_posts' => 1,
                'posts_per_page' => 20, 
                'post_type' => "product"
            ) );
            
            if( $search_results->have_posts() ) {
                while( $search_results->have_posts() ) {
                    $search_results->the_post();	
                    $return[] = array( 
                        "id" => $search_results->post->ID, 
                        "text"  => $search_results->post->post_title . " (ID:" . $search_results->post->ID . ")"
                    ); 
                }
            }
wp_send_json(["results" => $return]);

and its work,
but how to keep, result after i choose and it always in there when i click another.

thanks.

**Comment by sohan5005** [comment]
Have you ever find a solution to this issue?
